src/core/ngTableParams.ts

```diff
@@ -697,7 +697,10 @@ export function ngTableParamsFactory<T>(
                 more: 'More Pages', // default values
                 next: 'Next Page', // default values 
                 prev: 'Previous Page', // default values
-                pageNumPrefix: 'Page'
+                pageNumPrefix: 'Page', // default values
+                sortDescriptionAsc: 'sort in ascending order', // default values
```

Comment:
Can we translate these strings?

Reply:
Yes, in settings object there is a property called "translations" that when parsed with the correct properties replaces the property values in the accessibilityOptions object. This "translations" property can be parsed in when the table is instantiated on the frontend.